aws/logging.go

```diff
@@ -0,0 +1,68 @@
+// Code generated by aws/logging_generate.go DO NOT EDIT.
+package aws
+
+// ClientLogMode represents the logging mode of SDK clients. The client logging mode is a bit-field where
+// each bit is a flag that describes the logging behavior for one or more client components.
+// The entire 64-bit group is reserved for later expansion by the SDK.
+//
+// Example
+//  clientLogMode := aws.LogRetries | aws.LogRequest
+type ClientLogMode uint64
```

Comment:
minor nit, maybe also include example with adding a log level to an existing `clientLogMode` value.

```go
clientLogMode |= aws.LogRequest
```